concourse/scripts/self-update-pipeline.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Required variables are:
+# - DEPLOY_ENV
+# - MAKEFILE_ENV_TARGET
+# - SELF_UPDATE_PIPELINE
+#
+# Optional variables:
+# - BRANCH
+# - PIPELINES_TO_UPDATE
+
+set -u
+set -e
+
+if [ ! -d "./paas-cf" -o ! -f "concourse-manifest/concourse-manifest.yml" ]; then
+	echo "Resources paas-cf and concourse-manifest must be checkout"
```

Comment:
The indentation here is inconsistent with other scripts (we're using 2 spaces elsewhere).